student.rb

```diff
@@ -6,6 +6,12 @@ def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     @classroom = classroom
   end
 
+  def classroom=(classroom)
+    @classroom = classroom
+
+    @classroom.students.push(self) unless classroom.students.include?(self)
```

Comment:
- [ ] You shoud push the student to the `classroom` rather than the instance variable `@classroom`:
```rb
def classroom=(classroom)
  @classroom = classroom
  classroom.students.push(self) unless classroom.students.include?(self)
end
```
Check out [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) to learn more about the both-ways relationship.